app/mailers/decidim/friendly_signup/confirmation_codes_mailer.rb

```diff
@@ -17,6 +18,16 @@ def confirmation_instructions(user, opts)
           mail(to: "#{user.name} <#{@email}>", subject: I18n.t("decidim.friendly_signup.confirmation_codes.mailer.subject", organization: @organization.name, code: @code))
         end
       end
+
+      private
+
+      def confirm_path_url
+        "#{root_url}#{decidim_friendly_signup.confirmation_codes_path(confirmation_token: @token)}"
+      end
+
+      def root_url
+        @root_url ||= decidim.root_url(host: @user.organization.host)[0..-2]
```

Comment:
Hello, I am not sure to understand why we should remove last chars ?

Reply:
Is it to remove the last `/` of the `decidim.root_url` ?

Reply:
Actually if we only print this part of the code
```decidim.root_url(host: @user.organization.host)``` we will have at the end a ```/```and a space after it.
To take rid of that, we suppress the two last chars of the url which could have destroy the url if we had put a route after it (the route of friendly_signup confirm)

Reply:
<img width="306" alt="Capture d’écran 2022-11-08 à 10 07 32" src="https://user-images.githubusercontent.com/90462045/200522389-8bd99c91-fccb-460b-8206-fc4c5668f01e.png">

Here is the preview of what it looked like if we didn't get rid of the last chars

Reply:
Alright, I am just surprised to see a trailing whitespace returned by `decidim.root_url`, I think it is more safe to strip the trailing whitespaces rather than arbitrary remove 2 chars 

```suggestion
        @root_url ||= decidim.root_url(host: @user.organization.host).strip[0..-1]
       
```

Reply:
Because it ends with '&nbsp' char we can't use `strip`